docs/user-guide/README.md

```diff
@@ -25,7 +25,7 @@ see below.
 
 ## Older user guides
 
-* [HNC v0.6](https://github.com/kubernetes-sigs/multi-tenancy/tree/hnc-v0.6/incubator/hnc/docs/user-guide)
-* [HNC v0.5](https://github.com/kubernetes-sigs/multi-tenancy/tree/hnc-v0.5/incubator/hnc/docs/user-guide)
-* [HNC v0.4](https://github.com/kubernetes-sigs/multi-tenancy/tree/hnc-v0.4/incubator/hnc/docs/user-guide)
+* [HNC v0.6](https://github.com/kubernetes-sigs/hierarchical-namespaces/tree/hnc-v0.6/docs/user-guide)
+* [HNC v0.5](https://github.com/kubernetes-sigs/hierarchical-namespaces/tree/hnc-v0.5/docs/user-guide)
+* [HNC v0.4](https://github.com/kubernetes-sigs/hierarchical-namespaces/tree/hnc-v0.4/docs/user-guide)
```

Comment:
On the old branches, we haven't changed the directory structure and I'm not sure if we should. Perhaps we can just leave these as pointers back to the old repo? I'd actually be open to fully deleting the old branches from the new repo since none of them will build properly anyway.

Reply:
I deleted all the old branches, lets just hope we don't have to do any security patches on the old ones :)

Reply:
I think we're good :D Plus if we really, really needed to we could do it in the old repo.